exporter/zipkinexporter/config_test.go

```diff
@@ -61,6 +62,8 @@ func TestLoadConfig(t *testing.T) {
 					TLSSetting: configtls.TLSClientSetting{
 						InsecureSkipVerify: true,
 					},
+					MaxIdleConns:    httpDefaultClientSettings.MaxIdleConns,
+					IdleConnTimeout: httpDefaultClientSettings.IdleConnTimeout,
```

Comment:
You should set some values in the test file for these as well; usually the second test is with all values set.

Reply:
done, added some